engine/install/binaries.md

```diff
@@ -202,7 +202,7 @@ To install client binaries, perform the following steps:
 >
 > The following section describes how to install the Docker daemon on Windows
 > Server which allows you to run Windows containers only. The binaries for
-> Windows do not contain Docker components such as `buildx`, `docker scan`, and
+> Windows do not contain Docker components such as `buildx` and
```

Comment:
```suggestion
> Windows containers do not contain Docker components such as `buildx` and
```

?

Reply:
Leaving this suggestion out since the sentence refers to the daemon installation on a Windows host. But the whole paragraph could be improved to clarify this. I'll give it a go.